docsrc/allblocks.nim

```diff
@@ -75,6 +93,11 @@ nimibCode:
   nbCapture:
     echo "Captured!"
 
+nbCodeBlock: "nbTextWithCode"
+nbText: """
+`nbText` only stores the string it is given, but it doesn't store the code passed to `nbText`. For example, `nbText: fmt"{1+1}"` only stores the string `"2"` but not the code `fmt"{1+1}"` that produced that string. `nbTextWithCode` works like `nbText` but it also stores the code in the created block. It can be accessed with `nb.blk.code` right after the `nbTextWithCode` call. See the end of `numerical.nim` for an example.
```

Comment:
Once again, good explanation of why the block exists and differs from `nbText`. It would be nice if the `numerical.nim` could be a link to that page :D.

Reply:
This was your explanation: https://github.com/pietroppeter/nimib/pull/203#discussion_r1153644530
I do not get what would be the link in that case.

Reply:
Ok then I was lazy 😛 the link should be to https://pietroppeter.github.io/nimib/numerical.html but it's better if we use a relative link.

Reply:
What about: https://github.com/pietroppeter/nimib/blob/main/docsrc/numerical.nim ?
```suggestion
`nbText` only stores the string it is given, but it doesn't store the code passed to `nbText`. For example, `nbText: fmt"{1+1}"` only stores the string `"2"` but not the code `fmt"{1+1}"` that produced that string. `nbTextWithCode` works like `nbText` but it also stores the code in the created block. It can be accessed with `nb.blk.code` right after the `nbTextWithCode` call. See the end of [numerical.nim](https://github.com/pietroppeter/nimib/blob/main/docsrc/numerical.nim) for an example.
```

Reply:
I think it makes more sense to link to the actual webpage, as it contains both what's rendered and the code (under the `View Source` button). Change the link name to just `numerical` if that makes more sense.

Reply:
I changed the link to the one you proposed. I fear that we can not make a relative link.

Reply:
> I fear that we can not make a relative link.

Huh, why? 🤔

Reply:
Doesn't linking to `./numerical.html` work?

Reply:
I was expecting the link to be relative to the script directory rather than the output directory. I changed all three links to relative ones.